pkg/kev/converter/kubernetes/transform.go

```diff
@@ -166,12 +185,24 @@ func (k *Kubernetes) Transform() ([]runtime.Object, error) {
 					return nil, err
 				}
 				objects = append(objects, np)
+				renderedNetworkPolicy = np
 			}
 		}
 
 		allobjects = append(allobjects, objects...)
 	}
 
+	if renderedNetworkPolicy != nil {
+		stepNetworkPolicies := sg.Add("Creating networking policies")
+		stepNetworkPolicies.Success("Created networking policies")
```

Comment:
It seems we use the same text for a step and then when we update the status to success. Would it make sense to just define it once? Or should we add a suffice to that message when Success / Warn / Error occurs?